tests/foreman/ui/test_org.py

```diff
@@ -249,7 +249,11 @@ def test_positive_delete_bz1225588(self, org_name):
             # So switching to Default Org and then deleting.
             session.nav.go_to_select_org('Default Organization')
             self.org.remove(org_name)
-            self.assertIsNone(self.org.search(org_name))
+            for _ in range(5):
+                status = self.org.search(org_name)
+                if status is None:
+                    break
+            self.assertIsNone(status)
```

Comment:
I would call this `a workaround`, not a fix.
There is BZ for this behavior opened: [1251926](https://bugzilla.redhat.com/show_bug.cgi?id=1251926)
So I would suggest to use `bz_bug_is_open('1251926')` as a conditional for the loop.
Otherwise (bug is not opened anymore) I would keep the standard search in place.